### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+# Page Control for ASP.NET Web Forms - How to create a wizard interface
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3050/)**
 <!-- run online end -->
 
+This example demonstrates how to use the [Page Control](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPageControl) to create a wizard interface.
+
+![Create a Wizard](create-a-wizard.png)
+
+In the example, a step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays several fields. Mandatory fields are marked with an asterisk. Fill in the fields and click the **Next** button at the bottom of the current page to proceed to the next step. Click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > In the example, **a step** is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays **several** fields.

**First**. I understand that you intended to say that "any step is a tab page", which is correct. However, I feel a lack of clarity in this wording. What is important? The important thing that we speak of a wizard step. I believe we should explicitly mention this. The second important thing that every wizard step is a tab page in this example.
**Second**. Several. Several is a colloquialism, and we do not use colloquialisms in documentation. And no, "several" is not fewer than "multiple". The both words mean "two or more" in this context. Therefore, we use "multiple" everywhere we would have placed "several". No exceptions.
**Third**. To satisfy my curiosity: I should fill in all the fields on the current tab page to proceed further. Yet at the same time you say that I can click any of the named tabs to navigate to the corresponding step directly. Do I have to fill in all fields and pass validation before clicking a tab? Will a click on a different tab have any effect if I omitted a mandatory field on the current tab? The last sentence raises more questions than answers. There is a potential logical issue or lack of important information on how this example functions.

For now, I suggest the following edit, however, I cannot rectify the logical issue for you because I do not understand how this example works exactly.

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. Fill in the fields and click the **Next** button at the bottom of the current page to proceed to the next step. Click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```

Please clarify.

File: Readme.md [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
+
+Before you proceed to the next step, the [Validation Summary](https://docs.devexpress.com/AspNet/11626/components/data-editors/validationsummary) control validates values of the editors displayed on the current page and shows all validation errors in one block. Fix the errors to move forward.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > Before you proceed to the next step, the [Validation Summary](https://docs.devexpress.com/AspNet/11626/components/data-editors/validationsummary) control validates values of the editors displayed on the current page and **shows all validation errors in one block**.

What do you mean by "in one block"? From what I see on your image, the wizard form displays a list of validation errors below the **Next** button.

Again, please clarify.

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
**Structure and Semantics**

The code example description contains a potential logical flaw. I also do not see enough information on how the described example functions. A clarification is in order -- otherwise, the topic cannot serve its intended purpose.

**Images**

The image is clear and complies with DevExpress image guidelines. However, the image cannot rectify the semantic issues found in text.

**Language**

The language itself is good. I only updated one article case for the sake of clarity and replaced a colloquialism with a more formal word widely accepted throughout our documentation. Please never use "several". Use "multiple" everywhere, including all places where you would have used "several" as well.

**Bottom Line**

Please mind the integrity and unambiguity of information in your topics. The topic does not work for me in the current state. To my regret, I ended up with more questions than I had when started reading it. I'll be able to approve this pull request only once we get to the bottom of this. Thanks in advance.

File: Readme.md (hunk 1 above)
Comment (SvetlanaMikheeva) [reply Readme.md]: To move further, you must fill in all mandatory fields. But you can click the tab to go back, and in this case the validation is not performed, and the required fields may be empty.

Comment (dmitry-eliseev-devexpress) [reply Readme.md]: OK. Now I understand the way it works. Then we should explain exactly that and be done with it, don't you think?

Comment (SvetlanaMikheeva) [reply Readme.md]: Can we say the following?

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. You need to fill in the mandatory fields to move forward, but you can keep them empty to move backward. Click the **Next** button at the bottom of the current page to proceed to the next step, or click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```

Comment (dmitry-eliseev-devexpress) [reply Readme.md]: Yes. This makes more sense indeed.

"Backward" is a nominative form of a direction. When it comes to action, one moves backwards rather than backward.

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. You need to fill in the mandatory fields to move forward but you can keep them empty to move backwards. Click the **Next** button at the bottom of the current page to proceed to the next step, or click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.```

File: Readme.md (hunk 2 above)
Comment (SvetlanaMikheeva) [reply Readme.md]: A validation error can be displayed below the corresponding editor or within the Validation Summary. In this example, all errors are displayed within the Validation Summary that is located below the **Next** button. Maybe we should just omit "in one block".

```suggestion
Before you proceed to the next step, the [Validation Summary](https://docs.devexpress.com/AspNet/11626/components/data-editors/validationsummary) control validates values of the editors displayed on the current page and shows all validation errors. Fix the errors to move forward.
```

Comment (dmitry-eliseev-devexpress) [reply Readme.md]: Yes. This decision makes content more logically consistent. Thank you.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
We resolved all issues and are ready to move forward. Thank you.
